build.gradle.kts

```diff
@@ -22,6 +29,7 @@ dependencies {
     implementation("org.apache.maven:maven-model-builder:3.9.6")
     implementation("org.apache.maven:maven-model:3.9.6")
     implementation("com.google.guava:guava:32.1.3-jre")
+    implementation("com.android.tools.build:gradle:8.2.0")
```

Comment:
can this be provided? And we consume it from the classpath of the android project itself? I'm not sure I want this loaded into the classpath for non-android sbom projects.